src/modules/layerManager/components/LayerItem.js

```diff
@@ -75,9 +78,21 @@ export class LayerItem extends AbstractMvuContentPanel {
 				};
 			case Update_Layer_Collapsed:
 				return { ...model, layer: { ...model.layer, collapsed: data } };
+			case Update_Layer_Swipe:
+				return { ...model, ...data };
```

Comment:
Follow our usual pattern to update the model:
`case Update_Name_Of_Something:
  return {...model, specificProperty: data};`

This pattern helps us, to understand the flow of information from the extern sources (store or whatever) into the model and where the information sinks in.

Reply:
Done